branding/celo/src/brandingConfig.ts

```diff
@@ -15,3 +15,4 @@ export const CELO_SUPPORT_EMAIL_ADDRESS = 'support@celo.org'
 export const SUPERCHARGE_LEARN_MORE = 'https://valoraapp.com/support/supercharge-learn-more'
 export const INVITE_REWARDS_LEARN_MORE = 'https://valoraapp.com/support/invite-rewards-learn-more'
 export const DAPPS_LEARN_MORE = 'https://valoraapp.com/support/dapps-learn-more'
+export const SWAP_LEARN_MORE = 'https://valoraapp.com/support/swap-learn-more'
```

Comment:
[valora-app-branding](https://github.com/valora-inc/valora-app-branding) should be updated too as in https://github.com/valora-inc/valora-app-branding/commit/74d31b4513a868793c453a0d9c92e34314730a24.

Reply:
added https://github.com/valora-inc/valora-app-branding/pull/16